Example/Emission/stub_container.js

```diff
@@ -17,13 +17,29 @@ export default class StubbedContainer extends React.Component {
         getStoreData: () => {},
         primeCache: () => {}
       },
-    route: { name: 'string' }
+    route: { name: 'string' , params:{}, useMockData: true, queries: {}}
     };
   }
 
+  // TODO: Improve handling state, currently this works only when you full re-render the whole virtual view tree
+  // e.g. jumping between two stories of the same component will not trigger state changes. I gave a
+  // considerable amount of time, then noticed we _don't_ use too much state inside our components
+  // and have made the time tradeoff to say it's not worth it for now.
+
+  // If you are using state in a component, where you would normally set the default, you should check props first
+  // e.g. `this.state = this.props.state || { following: false, followersCount: props.artist.counts.follows };`
```

Comment:
The problem is that somehow we need to be able to pass state into a component that is _at minimum_ 2 levels down
`<StubbedContainer><RelayContainer><The Thing You Are Writing Your Story About>` - I explored a bunch of the API with 3/4 web devs over the course of the day, but eventually came to the conclusion that the only way to send any state for component being worked on, is via the props system.

I'm happy to call it something else, or say that you can't define `state` in a story, the thing was that when I reviewed more of the components we had, very few of them used and mutated the components `state`. It just happened to be the one I randomly picked first. So, it didn't feel worth trying to create something fully systematized.

The tradeoff is that you can only control _some_ of a component, in the examples in the header, it's trivial to change the artist name, birthday etc, but to make changes to the "following" state would be harder, which is why I added the default values via props.